### DISCUSSION TITLE
Add 8bitdo sn30 information in the wiki

### DISCUSSION THREAD

**Discussion by Ph0B1uS** [darthcloud/BlueRetro#674] [Q&A]
I had a problem with the dpad not working on an 8bitdo sn30 controller and as it turns out they have the same odd
misconfiguration as the n30 out of the box, the dpad is for some reason emulating a joystick.

This was resolved by holding select + up for a few seconds, this information is in the wiki for the n30 but can I ask if someone would please put the same information in there for the sn30 too.

Thanks in advance!

**Comment by darthcloud** [comment]
The sn30 got a joystick on it, that sound weird that the dpad would act as a joystick rather than dpad by default.

Good that you got yours working, but I don't think SN30/SF30 ship with that config by default so I rather not confuse people.

**Comment by Ph0B1uS** [comment]
I just bought two SN30s and both of them were configured as joysticks by default.While my sample rate is a bit small I did purchase them 2-3 months ago so I think keeping a note in the wiki might be a good idea and it’s not like you can break anything by configuring them to act as actual dpads either.I don’t know about the SF20 as I do not own any though.On 5 May 2023, at 14:17, Jacques Gagnon ***@***.***> wrote:﻿
The sn30 got a joystick on it, that sound weird that the dpad would act as a joystick rather than dpad by default.
Good that you got yours working, but I don't think SN30/SF30 ship with that config by default so I rather not confuse people.
